File: .sdk/tm/ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { ExchangeRatesSDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await ExchangeRatesSDK.test()
    equal(null !== testsdk, true)
  })

})

File: rb/test/exists_test.rb
# ExchangeRates SDK exists test

require "minitest/autorun"
require_relative "../ExchangeRates_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = ExchangeRatesSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
